webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -959,23 +992,30 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
       * @param resourceIris   the resources to query for.
       * @param versionDate    if defined, requests the state of the resources at the specified time in the past.
       * @param requestingUser the the client making the request.
+      * @param schemaOptions  the schema options submitted with the request.
       * @return a [[ReadResourcesSequenceV2]].
       */
     private def getResourcesV2(resourceIris: Seq[IRI],
                                propertyIri: Option[SmartIri] = None,
                                versionDate: Option[Instant] = None,
+                               targetSchema: ApiV2Schema,
+                               schemaOptions: Set[SchemaOption],
                                requestingUser: UserADM): Future[ReadResourcesSequenceV2] = {
 
         // eliminate duplicate Iris
         val resourceIrisDistinct: Seq[IRI] = resourceIris.distinct
 
+        val queryStandoff = SchemaOptions.queryStandoffWithTextValues(targetSchema = targetSchema, schemaOptions = schemaOptions)
```

Comment:
Please add a comment that this finds out whether standoff has to be queried and that this passed to `makeTextValueContentV2`.

Reply:
OK.

Reply:
Done in fe74ba6.